LPS-67704

daledotshan opened the pull request.

liferay-continuous-integration commented:
<h1>Build has started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/550/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-20.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/550/jenkins-report.html">here</a>.</b></p>

daledotshan commented:
information from Mohit
Right now if staging is enabled and Event is created on Staging then Invitation mail is sent to Resource creator, at this moment of time event is not yet published and not available to Live users (including the one who create resource) so logically, mail should not be triggered.

So i have added a logic that if group is staging then do not trigger any email.
I have added the logic in sendNotification as it is the main method where emails are triggered so if someone uses this method in future the logic will automatically apply to this.

daledotshan commented:
Hey Hugo,
This issue a little urgent please have to review it

Thanks,
Dale

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 2 hours 29 minutes 38 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/27d8ddb9d04ac13e3d744e57886ec236686f87a6">27d8ddb9d04ac13e3d744e57886ec236686f87a6</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/550/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32995/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30920/">test-portal-acceptance-pullrequest-batch(master)/functional-jboss64-mysql56-jdk8/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30921/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30922/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32996/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32997/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mariadb100-jdk8/2</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30924/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31573/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30926/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/10</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15029/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/11</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14916/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/12</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31575/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/13</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31237/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/14</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31988/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/15</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15028/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/16</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31941/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/17</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33000/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/18</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14917/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/19</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30923/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/2</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/29785/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/20</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31788/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/21</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30927/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/22</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15030/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/23</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15112/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/24</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31989/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/25</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31238/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/26</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31942/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/27</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31576/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/28</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30354/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/29</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32998/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/3</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31239/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/30</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32007/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/31</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31943/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/32</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28287/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/33</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30356/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/34</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28286/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/35</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32544/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/36</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31240/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/37</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32008/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/38</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30355/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/39</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31574/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/4</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28288/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/40</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31578/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/41</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31944/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/42</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32009/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/43</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31241/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/44</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33023/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/45</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31577/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/46</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32545/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/47</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28289/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/48</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30357/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/49</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31987/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/5</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14915/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/6</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32999/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31940/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30925/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk8/9</a></li><li><strike><strong><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32011/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/0</a></strong></strike></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31242/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31243/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/1</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31006/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-postgresql94-jdk8/2</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32012/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-sybase16-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15191/">test-portal-acceptance-pullrequest-batch(master)/functional-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30358/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32546/">test-portal-acceptance-pullrequest-batch(master)/integration-db2105-jdk8</a></li><li><strike><strong><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31945/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk8</a></strong></strike></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31007/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk8</a></li><li><strike><strong><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30359/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle12-jdk8</a></strong></strike></li><li><strike><strong><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15192/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk8</a></strong></strike></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31244/">test-portal-acceptance-pullrequest-batch(master)/integration-sybase160-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32013/">test-portal-acceptance-pullrequest-batch(master)/lpkg-override-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15032/">test-portal-acceptance-pullrequest-batch(master)/lpkg-persistence-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28290/">test-portal-acceptance-pullrequest-batch(master)/lpkg-startup-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15031/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><strike><strong><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32377/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2105-jdk8</a></strong></strike></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32014/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/14918/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk8</a></li><li><strike><strong><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31990/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle12-jdk8</a></strong></strike></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31946/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31789/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase160-jdk8</a></li><li><strike><strong><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15193/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></strong></strike></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28291/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32547/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30360/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31008/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15033/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30928/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/29786/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/24/">test-portal-acceptance-pullrequest-dist(master)/test-portal-acceptance-pullrequest-dist(master)-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/378/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-20.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/550/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/550/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>82 Jobs Passed.<br />8 Jobs Failed.</p><pre><code>Completed with the status of FAILURE.</code></pre></li><li><h5 job-result="FAILURE"><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/32011/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-oracle12-jdk8/0</a></h5><h6>Job Results:</h6><p>0 Tests Passed.<br />1 Test Failed.</p><ol><li><strong><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/32011/">test-portal-acceptance-pullrequest-batch&#40;master&#41; ?? 0,!master #32011</a></strong><pre><code>     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-batch\.xml:237: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-batch\.xml:594: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-tomcat\.xml:48: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\-tomcat\.xml:56: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:8489: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:2967: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:3063: The following error occurred while executing this line:
     \[echo\] /opt/dev/projects/github/liferay\-portal/build\-test\.xml:3481: No server is available at http://localhost:8080/web/guest
      \[get\] Getting: http://test\-1\-18/job/test\-portal\-acceptance\-pullrequest\-batch\(master\)/AXIS\_VARIABLE=0,label\_exp=\!master/32011//consoleText
      \[get\] To: /opt/dev/projects/github/liferay\-portal/20160826033555699\.txt
   \[delete\] Deleting: /opt/dev/projects/github/liferay\-portal/20160826033555699\.txt
     \[exec\] 26\-Aug\-2016 10:35:55\.889 INFO \[main\] org\.apache\.catalina\.core\.StandardServer\.await A valid shutdown command was received via the shutdown port\. Stopping the Server instance\.
     \[exec\] 26\-Aug\-2016 10:35:55\.890 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.pause Pausing ProtocolHandler \["http\-nio\-8080"\]
     \[exec\] 26\-Aug\-2016 10:35:55\.941 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.pause Pausing ProtocolHandler \["ajp\-nio\-8009"\]
     \[exec\] 26\-Aug\-2016 10:35:55\.991 INFO \[main\] org\.apache\.catalina\.core\.StandardService\.stopInternal Stopping service Catalina
     \[exec\] 26\-Aug\-2016 10:35:55\.992 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.stop Stopping ProtocolHandler \["http\-nio\-8080"\]
     \[exec\] 26\-Aug\-2016 10:35:55\.994 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.stop Stopping ProtocolHandler \["ajp\-nio\-8009"\]
     \[exec\] 26\-Aug\-2016 10:35:55\.994 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.destroy Destroying ProtocolHandler \["http\-nio\-8080"\]
     \[exec\] 26\-Aug\-2016 10:35:55\.994 INFO \[main\] org\.apache\.coyote\.AbstractProtocol\.destroy Destroying ProtocolHandler \["ajp\-nio\-8009"\]</code></pre></li></ol></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31945/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk8</a></h5><h6>Job Results:</h6><p>5357 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/31945//testReport/com.liferay.portal.search/SearchPaginationTest/testSearchWithResults">SearchPaginationTest.testSearchWithResults</a></li></ol></li><li><h5 job-result="ABORTED"><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/30359/">test-portal-acceptance-pullrequest-batch(master)/integration-oracle12-jdk8</a></h5><pre><code>Build was aborted</code></pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/15192/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk8</a></h5><h6>Job Results:</h6><p>5354 Tests Passed.<br />3 Tests Failed.</p><ol><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=2,label_exp=!master/15192//testReport/junit.framework/TestSuite/com_liferay_portal_dao_orm_SQLInstrTest">TestSuite.com.liferay.portal.dao.orm.SQLInstrTest</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=2,label_exp=!master/15192//testReport/com.liferay.portal.dao.orm.hibernate/DB2DialectTest/testPagingWithNegativeStartAndNegativeEnd">DB2DialectTest.testPagingWithNegativeStartAndNegativeEnd</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=2,label_exp=!master/15192//testReport/com.liferay.portal.kernel.portlet.bridges.mvc/MVCActionCommandTest/testSingleMVCActionCommand">MVCActionCommandTest.testSingleMVCActionCommand</a></li></ol></li><li>...</li></ol><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(master)/">here</a>.</h5>

hhuijser commented:
Sent to Adam Brandizzi for review: https://github.com/brandizzi/liferay-portal/pull/289
